packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -88,8 +87,8 @@ test.describe('severity-2 #smoke', () => {
       const config = await configPage.getConfig();
       const [email] = emails;
       test.fixme(
-        config.showReactApp.signUpRoutes,
-        'this test goes through the signup flow instead of sign in, skipping for react'
+        true,
+        'to be fixed to use signin and not signup flow, see FXA-8973'
```

Comment:
Same thing here:

```suggestion
        config.showReactApp.signUpRoutes,
        'this test goes through the signup flow instead of sign in, skipping for react, see FXA-8973'
```